### PR TITLE
Removing default alert style

### PR DESCRIPTION
**Pull request by bia**
See [ui-components/pull/#390comment](https://github.com/weaveworks/ui-components/pull/390#pullrequestreview-167414511).
Removing default alert style and changing unspecified styles to info.

### CODE REVIEW

File: src/components/Alert/example.js [hunk 1]
```diff
@@ -65,31 +60,6 @@ export default class AlertExample extends React.Component {
             Error: Wow you really screwed this up...
           </Alert>
         </Example>
-        <Example>
```
Comment (guyfedwards) [inline src/components/Alert/example.js]: Why are the multiline and multiline+title examples being removed? Seems separate to this PR?

Comment (guyfedwards) [reply src/components/Alert/example.js]: I couldn't find anywhere explicitly using the multiline feature so maybe fine to remove. I also couldn't find anywhere we use the title but I think having an example showing the title is worthwhile. 

Could maybe remove just the multiline and change the title one to be a `warning` or `error` to align with the warning icon?

Comment (rndstr) [reply src/components/Alert/example.js]: > and didn't want people to take that as an example of something to do

Are we saying multiline alerts should not be used?

+1 on title/icon.

File: src/theme/misc.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { colors } from './colors';
 
 export const overlayIconSize = '300px';
-export const textColor = colors.black;
+export const textColor = colors.white;
```
Comment (guyfedwards) [inline src/theme/misc.js]: This seems like a fairly large change? The snapshot above suggests that the `Text` component now has `color: hsl(0,0,100);` which seems like an issue?

Comment (bia) [reply src/theme/misc.js]: My mistake! Didn’t realise how big of a change that would be.

**Comment by bia** [comment]
>Are we saying multiline alerts should not be used?

Yes, if possible.

>I think the Default example could also be removed, the fact that we default to Info style should be obvious from the table above.

It was already - screenshot was just not up to date.
Now it looks like
screenshot 2018-10-25 at 15 17 29

**Pull request merged**